src/pages/PersonaFamily/PersonaFamily.tsx

```diff
@@ -0,0 +1,239 @@
+import React, { FC, useEffect } from 'react';
+import { TopNav } from 'components/TopNav/TopNav';
+import { WideButton } from 'components/Button/WideButton';
+import { Stepper } from 'components/Stepper';
+import { useTranslation } from 'react-i18next';
+import styled from 'styled-components';
+import { APP_ROUTES } from '../../global/AppRouter/routes';
+import { useHistory } from 'react-router-dom';
+import { InputWithSuffixIcon } from '../../components/InputWithSuffixIcon/InputWithSuffixIcon';
+import EmailIconSvg from 'assets/email.svg';
+
+const MainComponent = styled.div`
```

Comment:
Extract this form component as a separate one because we can reuse it in the future in other project.